setup.py

```diff
@@ -65,11 +65,11 @@ def get_long_description() -> str:
           'Programming Language :: Python :: 3.7',
           'Programming Language :: Python :: 3.8',
           'Programming Language :: Python :: 3.9',
-          'Programming Language :: Python :: 3.10'],
+          'Programming Language :: Python :: 3.10',
+          'Programming Language :: Python :: 3.11'],
       license='MIT',
-      install_requires=['spirack>=0.1.8', 'numpy>=1.20', 'serialize', 'pymongo', 'zhinst;python_version<"3.10"',
-                        'requests', 'qcodes==0.33.0', 'qcodes_contrib_drivers==0.13.1', 'dataclasses-json',
-                        'zhinst==22.2.26577'],
+      install_requires=['spirack>=0.1.8', 'numpy>=1.20', 'serialize', 'pymongo',
+                        'requests', 'qcodes==0.33.0', 'qcodes_contrib_drivers==0.13.1', 'dataclasses-json'],
```

Comment:
I can remember we pinned qcodes and qcodes_contrib_drivers because of ZI incompatibilities. I think we can say >= instead of == for both.